Dangerous uplink tweaks & Shuriken changes

Kell-E opened the pull request:
Adds a box of six shuriken to the uplink for 18 TC

Reduces the throw speed of shuriken from 10 to 8 (Prevents wall splat / impalement)
Increases the throw force from 15 to 20 to make up for the lost damage via speed
Reduces the size of shuriken from pistol size to syringe size (Medium to tiny, 3 -> 1)

Adds a katana to the uplink for 24 TC 
Reduces the cost of the pike cube from 70 to 44, 70 was excessive while at a price point of 44 it's only possible to purchase two, or three with a radio. 

Removes the zipgun and replica pistol from the uplink
The zipgun wasn't ever used and the replica pistol was roughly equal in strength to a 40 TC weapon without any notable downsides.

Modifies the cost of merc railguns to 65/75 respectively, both were never used because of their significant price compared to what they actually offered in return. They *might* end up being pretty strong but I'd rather they got used and we made adjustments later than have them soft removed via excessive prices.

🆑 Kell-E
rscadd: Adds a box of shurikens to the uplink for 18TC.
balance: reduces the size of shurikens from medium to tiny (Pistol -> syringe size), removes shurikens ability to gib limbs and wall splat but increased their throw force from 15 to 20.
rscadd: Adds a katana to the uplink for 24 TC.
balance: reduces the cost of the pike cube from 70 to 44.
rscdel: Removes the replica pistol and zipgun from the uplink.
balance: Reduces the cost of the two merc railguns from 109/130 to 65/75 respectively.
/🆑

SierraKomodo commented on code/datums/uplink/highly_visible_and_dangerous_weapons.dm:
```diff
@@ -4,23 +4,17 @@
-	path = /obj/item/gun/projectile/pistol/throwback
+/datum/uplink_item/item/visible_weapons/shuriken
+	name = "Box of shurikens"
+	desc = "A small box with six shuriken, notably dangerous"
```
```suggestion
	desc = "A small box with six shuriken, notably dangerous."
```

SierraKomodo replied:
Minor nitpick but still, grammar :P

Kell-E committed:
Dangerous uplink tweaks & Shuriken changes

Adds a box of six shuriken to the uplink for 18 TC

Reduces the throw speed of shuriken from 10 to 8 (Prevents wall splat / impalement)
Increases the throw force from 15 to 20 to make up for the lost damage via speed
Reduces the size of shuriken from pistol size to syringe size (Medium to tiny, 3 -> 1)

Adds a katana to the uplink for 24 TC 
Reduces the cost of the pike cube from 70 to 44, 70 was excessive while at a price point of 44 it's only possible to purchase two, or three with a radio. 

Removes the zipgun and replica pistol from the uplink
The zipgun wasn't ever used and the replica pistol was roughly equal in strength to a 40 TC weapon without any notable downsides.

Modifies the cost of merc railguns to 65/75 respectively, both were never used because of their significant price compared to what they actually offered in return. They *might* end up being pretty strong but I'd rather they got used and we made adjustments later than have them soft removed via excessive prices.


I am dumb

:)
changes based upon review
